### PR TITLE
Add initial repo layout with CCM prototype

### PR DESCRIPTION
**Pull request by frapposelli**
Create scaffolding for repo, with initial CCM prototype (thanks to cloud-provider-openstack)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+
+**Anything else we need to know?**:
+
+**Environment**:
+- openstack-cloud-controller-manager version:
```
Comment (dougm) [inline .github/ISSUE_TEMPLATE.md]: s/openstack/vsphere/

Comment (frapposelli) [reply .github/ISSUE_TEMPLATE.md]: good catch! 😄

**Pull request merged**